cellmlmanip/parser.py

```diff
@@ -152,8 +152,8 @@ def _add_units(self, model):
         units_elements = model.findall(Parser.with_ns(XmlNs.CELLML, 'units'))
 
         # get list of built-in cellml units
-        from cellmlmanip.units import CELLML_UNITS
-        units_found = set(CELLML_UNITS)
+        from cellmlmanip.units import _CELLML_UNITS
```

Comment:
It's a small point, I'm happy to change it if you like!
I tend to think of lists like these as implementation details rather than part of the public API. Not sure you could break things by modifying it, but you could certainly change the expected behaviour

Reply:
I was mainly thinking it might be useful to users as a read-only property. But actually you can get the info more usefully just by creating an empty `UnitStore`, so having this private makes more sense.